homeassistant/components/renault/strings.json

```diff
@@ -174,7 +174,7 @@
     },
     "ac_cancel": {
       "name": "Cancel A/C",
-      "description": "Canceles A/C on vehicle.",
+      "description": "Cancels A/C on vehicle.",
```

Comment:
This could have been a separate bugfix PR 😉 but I think it's ok to leave it in here.